### ISSUE TITLE
Support for Cupertino widgets

### ISSUE THREAD

**Issue by adario** [djarjo/flutter_input#3]
@djarjo Any plans to support Cupertino widgets?
At present, the library requires a Material hierarchy — it would be nice to remove this requirement, in order to use e.g. a CupertinoScaffold instead of a regular Scaffold.

**Comment by hlemcke** [comment]
Good point, although I have no idea how to create a package which dynamically selects Cupertino or Material widgets. Only widget working that way currently is `Switch.adaptive`. I leave this open trying to `adapt` by package specific code.

**Comment by adario** [comment]
@djarjo Perhaps you could use something akin to `PlatformWidget` from [flutter_platform_widgets](https://pub.dev/packages/flutter_platform_widgets) ?
